### DISCUSSION TITLE
eigen external link

### DISCUSSION THREAD

**Discussion by matt256** [microsoft/onnxruntime#8048] [Other Q&A]
Is the eigen submodule still properly linked? At least for me, it looks like that link isn't recognized and not cloning properly. Am I going crazy or is this an actual issue? Thanks in advance

https://github.com/microsoft/onnxruntime/tree/master/cmake/external

**Comment by snnn** [comment]
Because it is on gitlab, not github. So github's web UI doesn't show it. But if you access the code from command line, it should be fine.

**Reply by matt256** [reply]
thank you for clarifying and helping. I'm a bit new to this stuff.
